drupal.make

```diff
@@ -3,10 +3,11 @@ api = 2
 
 ; Core
 projects[drupal][type] = core
-projects[drupal][version] = 7.54
-projects[drupal][patch][] = http://drupal.org/files/issues/menu-get-item-rebuild-1232346-45.patch
-projects[drupal][patch][] = http://drupal.org/files/ssl-socket-transports-1879970-13.patch
-projects[drupal][patch][] = http://drupal.org/files/issues/translate_role_names-2205581-1.patch
+projects[drupal][version] = 7.56
```

Comment:
This is Drupal Core 7.56 - README mentions 7.55. Which is it? 😄

Reply:
Updated